connectors/sources/jira.py

```diff
@@ -104,7 +104,7 @@ def __init__(self, configuration):
         self.configuration = configuration
         self._logger = logger
         self.data_source_type = self.configuration["data_source"]
-        self.host_url = self.configuration["jira_url"]
+        self.host_url = self.configuration["jira_url"] + "/"
```

Comment:
Should we prune `/` in the end of the `self.configuration["jira_url"]` if present?

Reply:
`urljoin` will take care of it if the host url contains double slash at the end. However, we can add it only if the trailing slash is not present.